### PR TITLE
Check the first toilet code match to see if it is exact.

### PR DESCRIPTION
**Pull request by hodgestar**
Currently we rely on the toilet code API returning only one match when there is a toilet code that matches exactly. We could update the API to work this way but it seems a bit inconsistent -- one expects search to always return results with the best results first, not to limit results because there is one really good match.

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: test/app.test.js [hunk 1]
```diff
@@ -480,6 +480,16 @@ describe("App", function() {
                 .run();
         });
 
+        it("should detect the match even if case differs", function() {
+            return tester
+                .setup.user.lang('en')
+                .input('MN34')
```
Comment (rudigiesler) [inline test/app.test.js]: Shouldn't this be lower case?

Comment (hodgestar) [inline test/app.test.js]: Good catch.

**Comment by hodgestar** [comment]
Test fixed. Ready for re-review.

**Comment by rudigiesler** [comment]
:+1:

**Pull request closed without merging**